src/pdm/cli/commands/python.py

```diff
@@ -136,7 +133,7 @@ def install_python(project: Project, request: str) -> PythonInfo:
         version, _, arch = version.partition("-")
         arch = "x86" if arch == "32" else (arch or THIS_ARCH)
 
-        ver, python_file = get_download_link(version, implementation=implementation, arch=arch)
```

Comment:
made it more obvious that build_dir is used and is False, like with the new util method `get_all_installable_python_versions`